src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+ul,
+body,
+nav,
+a,
+html {
+  text-decoration: none;
   margin: 0;
+  padding: 0;
+  list-style: none;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: black;
+  text-transform: uppercase;
```

Comment:
Use text transform only for navigation link class, it's redundant for the entire page